doc/guides/test/ztest.rst

```diff
@@ -341,3 +341,22 @@ expect the values ``a=2`` and ``b=3``, and telling ``returns_int`` to return
 
 .. doxygengroup:: ztest_mock
    :project: Zephyr
+
+Customizing Test Output
+***********************
+The way output is presented when running tests can be customized.
+An example can be found in :zephyr_file:`tests/ztest/custom_output`.
+
+Customization is enabled by setting :option:`CONFIG_ZTEST_TC_UTIL_USER_OVERRIDE` to "y".
+
+Then, a file :file:`tc_util_user_override.h` is expected to be found by Zephyr.
```

Comment:
Combine these two sentences for clarity:
```
Then, create a file named :file:`tc_util_user_override.h` with your customizations.
```

Reply:
I reordered the paragraph slightly to hopefully increase clarity.